### ISSUE TITLE
More baudrate options

### ISSUE THREAD

**Issue by user95165435** [jzhvymetal/WiresharkSerialAdapter#1]
Hello,

could you do a version with baudrate 230400? Or rather allow it to be textbox too so user can add what ever value they want. 

We attempted to use this tool to test one system, but there the speed is fixed to 230400.  Getting everything needed to compile this was not so straightforward so I thought to ask if you would have time to update this :)

**Comment by jzhvymetal** [comment]
This should be an easy fix, but my build environment changed. It has something to do with VS Code/Mingw building as 64-bit when before it was 32-bit built. All the code is using strings instead of wstrings. Once I fix my build environment, I will release and update. Also, I will provide an update on how to create a build environment.

Just note that not all serial adapters support changing the baud rate to all rates. Make sure your adapter can do it by using a simple terminal program.

**Comment by jzhvymetal** [comment]
I updated the code for baud 230400. Wireshark Extcap has an option to allow a type "editseletor" that allows selecting from a list or free typing. For some reason, it is not working when I try to use it. I assume that it works just like a type "selector". When I use the type "editseletor", it will not show the option in the settings dialog.

Also, I updated the readme file with instructions for compilation. The code has also been updated so it will not have build errors when compiling for 64-bit.
